### PR TITLE
add readme template

### LINKED ISSUE #3: README template
Please provide README template for each new examples :)

### PR DESCRIPTION
**Pull request by Kenji-H**
### Summary
- adde readme template
- update readme

### Issues
- resolve #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
 
-## example summary
+This project aims to collect know-hows and best practices about Apache Beam.
+
+## Example summary
 |  example  |  description  |
 | ---- | ---- |
 |  [word-count-beam](word-count-beam)  |  This is most likely your first Apache Beam project.   It includes fundamental features and best practices of Beam.  |
 |  [multi-storage](multi-storage)  |  This shows how to push data to multiple storage locations.  |
+
```
Comment (byam) [inline README.md]: Could you add `gcs-triggered` here?

File: README-template.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### set up
+Please set environment variables as follows:
+
+- `RUNNER`: Beam Runner name. ex: DirectRunner, DataflowRunner
+- `GOOGLE_PROJECT`: Google Project name.
+- `PUBSUB_TOPIC`: Pub/Sub topic name.
+- `OUTPUT_FILE`: Output file path.
+
+Create a Pub/Sub topic.
```
Comment (byam) [inline README-template.md]: Create a topic, if you have not?

**Review by byam — CHANGES_REQUESTED** [review]
It would be better if examples are provided. For example you can write here `Word Count` scenario.

**Comment by Kenji-H** [comment]
@byam Thanks for your review. I updated the files.

**Review by byam — APPROVED** [review]
LGTM!

**Pull request merged**